packages/plugins/i18n/server/constants/iso-locales.json

```diff
@@ -1095,6 +1095,10 @@
     "code":"gl-ES",
     "name":"Galician (Spain) (gl-ES)"
   },
+  {
+    "code":"lao",
+    "name":"Lao (Laos) (lao)"
+  },
```

Comment:
Where do we get the name codes?
If it is from laos, it seems to be lao-LA, but don't know if we use any other reference
https://www.localeplanet.com/icu/lo-LA/index.html

Reply:
lao-LA would be more specific, if we wanted a Laos dialect of Lao.